packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -339,6 +339,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     } = this.props;
     const { isOpen, focused, suggestedDisplayValue } = this.state;
     this._currentVisibleValue = this._getVisibleValue();
+    const hasText = this._currentVisibleValue === '';
```

Comment:
Why are we keying off of `_currentVisibleValue` here? Wouldn't we only want to show the error message is  or not?

can't we 
```js
{ hasErrorMessage && <div> error message div </div> }
```

Reply:
You're right, thanks Micah! Don't need to make a new variable

Reply:
I just double checked it works the same: 
![image](https://user-images.githubusercontent.com/4161791/70174257-6475e500-1689-11ea-9c38-e4fc21af103b.png)